extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -101,11 +107,54 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
     return Services.storyStoreServiceForOrNull(this.win).then(
       (storeService) => {
         this.storeService_ = storeService;
-        this.initializeListeners_();
+        this.element.addEventListener('click', this.onClick_.bind(this), true);
+        this.storeService_.subscribe(
+          StateProperty.MUTED_STATE,
+          this.onMutedStateChange_.bind(this)
+        );
       }
     );
   }
 
+  /**
+   * When the sticker is clicked, toggle muted state and highlight
+   * the mute control on system layer.
+   * @private
+   */
+  onClick_() {
+    this.storeService_.dispatch(Action.TOGGLE_MUTED, false);
+
+    this.getSystemLayerEl_();
+    this.systemLayerEl_.classList.toggle(
+      'i-amphtml-story-highlight-mute-audio-control',
+      true
+    );
+  }
+
+  /**
+   * Disable highlight once the story gets muted again.
+   * @private
+   * @param {boolean} isMuted
+   */
+  onMutedStateChange_(isMuted) {
+    if (isMuted) {
+      this.getSystemLayerEl_();
+      this.systemLayerEl_.classList.toggle(
+        'i-amphtml-story-highlight-mute-audio-control',
+        false
+      );
+    }
+  }
+
+  /** @private */
+  getSystemLayerEl_() {
+    if (!this.systemLayerEl_) {
+      this.systemLayerEl_ = this.element
+        .closest('amp-story')
+        .querySelector('.i-amphtml-system-layer-host');
+    }
```

Comment:
Return the system layer element in this function so it makes more sense in the caller (`this.getSystemLayerEl_().classList.toggle(...)`). I don't think caching the result matters too much here, because users should only click on the mute button once; so the private variable is somewhat innecessary.

```suggestion
    return this.element.closest('amp-story').querySelector('.i-amphtml-system-layer-host');
```